.agents/skills/mutation-testing/SKILL.md

```
@@ -0,0 +1,67 @@
+
+```sh
+npm run test-mutation-file -- "src/cartesian/*.tsx"
+```
+
+Only skip `--checkers ""` (i.e. fall back to `npm run test-mutation -- --mutate <path>`) when you need type-safety verification on the surviving mutants, e.g. right before committing.
+
+## Checking results
+
+After a run, open the HTML report:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)